editor.planx.uk/src/pages/FlowEditor/lib/store/team.ts

```diff
@@ -108,6 +108,19 @@ export const teamStore: StateCreator<
             integrations {
               hasPlanningData: has_planning_data
             }
+            settings: team_settings {
+              boundaryUrl: boundary_url
+              boundaryBBox: boundary_bbox
+              referenceCode: reference_code
+              helpEmail: help_email
+              helpPhone: help_phone
+              helpOpeningHours: help_opening_hours
+              emailReplyToId: email_reply_to_id
+              homepage: homepage
+              externalPlanningSiteName: external_planning_site_name
+              externalPlanningSiteUrl: external_planning_site_url
+              submissionEmail: submission_email
+            }
```

Comment:
In order to re-use the existing `/download-application-files/:sessionId` endpoint, we need team settings like `submissionEmail` to be available in the editor when the team store is initialised - is this okay? Am I forgetting any specific reasons that settings were omitted from this query and _not_ initialised?

Reply:
I think this is cool, I remember I had a bit of trouble with the query, but think that was due to an issue on timing of changes with submissionEmail than anything code related that would stop this happening. All the settings should be defined in the database and able to be read, nothing gets generated after this... unless you edit a setting.